<launch>

<!---- joy node ---->
	<node respawn="true" pkg="joy" type="joy_node" name="joystick" >
	<param name="dev" type="string" value="/dev/input/js0"/>
	<param name="deadzone" value="0.12"/>
    
	<!-- Rate in Hz at which a joystick that has a non-changing state will resend the previously sent message (default=0)-->
	<param name="autorepeat_rate" value="60"/>
	<param name="coalesce_interval" value="0.001"/>
	</node>
	
<!---- axies ---->
	<param name="axis_linear" value="1" type="int"/>
	<param name="axis_angular" value="0" type="int"/>
	<param name="scale_linear" value="2" type="double"/>
	<param name="scale_angular" value="2" type="double"/>

<!---- remote_controller node ---->
	<node pkg="remote_controller" type="remote_controller_node" name="remote_controller"/>
</launch>
